docs/data/base/components/field/field.md

```diff
@@ -0,0 +1,185 @@
+---
+productId: base-ui
+title: React Field component and hook
+components: FieldRoot, FieldLabel, FieldMessage, FieldControl, FieldValidity
+githubLabel: 'component: field'
+packageName: '@base_ui/react'
+---
+
+# Field
+
+<p class="description">Fields represent an individual section of a form containing an associated control and label, as well as any description or validation messages.</p>
+
+{{"component": "@mui/docs/ComponentLinkHeader", "design": false}}
+
+{{"component": "modules/components/ComponentPageTabs.js"}}
+
+{{"demo": "UnstyledFieldIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
+
+## Installation
+
+Base UI components are all available as a single package.
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install @base_ui/react
+```
+
+```bash yarn
+yarn add @base_ui/react
+```
+
+```bash pnpm
+pnpm add @base_ui/react
+```
+
+</codeblock>
+
+Once you have the package installed, import the component.
+
+```ts
+import * as Field from '@base_ui/react/Field';
+```
+
+## Anatomy
+
+Fields are implemented using a collection of related components:
+
+- `<Field.Root />` is a top-level component that wraps all other components.
+- `<Field.Control />` renders the control when not using a native Base UI component.
+- `<Field.Label />` renders a label for the control.
+- `<Field.Message />` renders an optional message for the control to describe it or show validation errors.
+- `<Field.Validity />` is an optional render prop component that enables reading raw `ValidityState` to render custom JSX.
```

Comment:
"render prop component" - this is a bit confusing. I'd just say it's a component and in the underline in the Validitiy component section, that it can accept function as a child.